app/src/main/java/cm/aptoide/pt/home/BottomNavigationActivity.java

```diff
@@ -39,13 +41,15 @@
   @Override public void showFragment(Integer menuItemId) {
     Fragment currentFragment = getFragmentNavigator().getFragment();
     Fragment selectedFragment = null;
+    String defaultStoreName = ((AptoideApplication) getApplicationContext()).getDefaultStoreName();
     //Each view from the  BottomNavigation has to implement a scrollToTop method when clicked again (see BottomHomeFragment)
     //Each fragment should implement it's own action bar
     switch (menuItemId) {
       case R.id.action_home:
         selectedFragment = new BottomHomeFragment();
         break;
       case R.id.action_search:
+        selectedFragment = SearchResultFragment.newInstance("", defaultStoreName, true);
```

Comment:
Can the SearchResultFragment have a newInstance without search query where you would put empty by default?

Reply:
Fixed